src/rust/ide/lib/view/src/graph_editor/component/documentation_view.rs

```diff
@@ -0,0 +1,159 @@
+//! Documentation view visualization generating and presenting Enso Documentation under
+//! the documented node.
+
+use crate::prelude::*;
+
+use crate::graph_editor::component::visualization::*;
+use crate::graph_editor::component::visualization;
+
+use enso_frp as frp;
+use ensogl::display::DomSymbol;
+use ensogl::display::scene::Scene;
+use ensogl::display;
+use ensogl::system::web;
+use ensogl::system::web::StyleSetter;
+
+/// Generates Documentation View stylesheet.
+pub fn get_doc_style() -> String {
+    format!("<style>{}</style>", include_str!("documentation_view/style.css"))
+}
+
+#[derive(Clone,CloneRef,Debug)]
+#[allow(missing_docs)]
+pub struct DocumentationViewModel {
+    logger : Logger,
+    dom    : DomSymbol,
+    size   : Rc<Cell<Vector2>>,
+}
+
+impl DocumentationViewModel {
+    /// Constructor.
+    fn new(scene:&Scene) -> Self {
+        let logger = Logger::new("DocumentationView");
+        let div    = web::create_div();
+        let dom    = DomSymbol::new(&div);
+        let screen = scene.camera().screen();
+        let size   = Rc::new(Cell::new(Vector2(290.0,screen.height - 30.0)));
```

Comment:
hardcoded numbers. Remove or make them well documented constants in a constant section on top of the file